### ISSUE TITLE
Add Select-String -Raw parameter documentation for version 7

### ISSUE THREAD

**Issue by Jawz84** [MicrosoftDocs/PowerShell-Docs#4537]
Issue Details
-----------------
An extra switch `-Raw` will be added to Select-String in PowerShell 7.0.0-preview.2, see https://github.com/PowerShell/PowerShell/pull/9901. 

The implementation of the `-Raw` switch also introduced new parametersets. 

Version(s) of document impacted
------------------------------
- [x ] Impacts 7 document
- [ ] Impacts 6 document
- [ ] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not selecting all version of documents
----------------------------------------------------
- [x] The documented feature was introduced in selected version of PowerShell
- [ ] This issue only shows up in selected version of the document

**Comment by sdwheeler** [comment]
This makes `Select-String` act more like grep.

**Comment by Jawz84** [comment]
> This makes Select-String act more like grep.
Yes, it makes it output raw strings. If there is anything I can help do or clarify, let me know.
